### PR TITLE
Division function added

### PR DESCRIPTION
**Pull request by Prince-sud0**
1. Division function was added
2. Division button was added

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -24,6 +24,11 @@ function multiplication() {
     operator = "*"
     document.getElementById("display").value = currentInput
 }
+function division() {
```
Comment (Alea237) [inline script.js:28]: please add exrtra line

**Review by Alea237 — CHANGES_REQUESTED** [review]
please go through the comment and fix the code!

**Review by Prince-sud0 — COMMENTED** [review]
fixed as per the request

**Review by Alea237 — APPROVED** [review]

**Comment by Prince-sud0** [comment]
Thanks for the Experience Prof.

**Pull request merged**